update.sh

```diff
@@ -13,9 +13,11 @@ versions=( "${versions[@]%/}" )
 travisEnv=
 for version in "${versions[@]}"; do
 	latest=$(curl -sSL 'https://owncloud.org/changelog/' |tac|tac| grep -o -m 1 "\(Version\|Release\) ${version}.[[:digit:]]\+" | sed -rn 's/(Version|Release) (.*)/\2/p')
-	sed -ri -e 's/^(ENV OWNCLOUD_VERSION) .*/\1 '"$latest"'/' "$version/Dockerfile"
-	
-	travisEnv='\n  - VERSION='"$version$travisEnv"
+	travisEnv='\n  - VERSION='"$version"' VARIANT='"$travisEnv"
```

Comment:
Yes, that is how we want to tag the images, but this is just generating the travis test so it would build it twice for no benefit.  Tags and which Dockerfile(s) for the official builds are specified in a "library" file in [`docker-library/official-images`](https://github.com/docker-library/official-images/);  `generate-stackbrew-library.sh` generates that library file for this repo.